CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
-# Version X.XX.X (Unreleased)
+# Version 3.33.0 (2022-12-13)
 ### Added
 * Added SDK support for creating batches with up to 100k data rows
+* Added optional media_type to `client.create_ontology_from_feature_schemas()` and `client.create_ontology()`
+
+### Changed
+* String representation of `DbObject` subclasses are now formatted
 
```

Comment:
can add also 
```
* Common errors from data rows creation now explicitly shown in task.errors property`
```

Reply:
That's not a change tied to this specific version of the SDK. I think that's better communicated via our docs